### ISSUE TITLE
ISO: Boilerplate text should be in Metanorma-ISO instead of stylesheet

### ISSUE THREAD

**Issue by ronaldtse** [metanorma/mn-native-pdf#1]
These lines should refer to Metanorma XML content:

https://github.com/metanorma/mn-native-pdf/blob/91415b5e3813dc49e9bce804c2617d4e763889ed/xslt/mn-iso.xsl#L95-L116

This can only be done after the upstream issue is complete:
https://github.com/metanorma/metanorma-iso/issues/292

This is for @Intelligent2013

**Comment by opoudjis** [comment]
This is resolved now:

The copyright notice, starting with "© ISO 2019, Published in Switzerland.", appears now in iso-standard/boilerplate/copyright-statement; e.g.

````
<boilerplate>
  <copyright-statement>
    <clause>
    <p id="authority1">
    © ISO 2019, Published in Switzerland
  </p>

  <p id="authority2">
    All rights
reserved. Unless otherwise specified, no part of this publication may be
reproduced or utilized otherwise in any form or by any means, electronic or
mechanical, including photocopying, or posting on the internet or an intranet,
without prior written permission. Permission can be requested from either ISO
at the address below or ISO's member body in the country of the requester.
  </p>

  <p id="authority3" align="left">
    ISO copyright office<br/>
    Ch. de Blandonnet 8 • CP 401<br/>
    CH-1214 Vernier, Geneva, Switzerland<br/>
    Tel.   + 41 22 749 01 11<br/>
    Fax   + 41 22 749 09 47<br/>
    copyright@iso.org<br/>
    www.iso.org
  </p>
</clause>
  </copyright-statement>
````

The "Warning for WDs and CD" clause, which is optional, appears now in iso-standard/boilerplate/license-statement; e.g.

````
<boilerplate>
...
<license-statement>
    <clause>
<title>Warning for WDs and CDs</title>

<p>This
document is not an ISO International Standard. It is distributed for review and
comment. It is subject to change without notice and may not be referred to as
an International Standard.</p>

<p>Recipients
of this draft are invited to submit, with their comments, notification of any
relevant patent rights of which they are aware and to provide supporting
documentation.</p>
</clause>
  </license-statement>

</boilerplate>
````

**Comment by Intelligent2013** [comment]
I've fixed the issue in mn-iso.xsl.

**Comment by ronaldtse** [comment]
@Intelligent2013 the documents have been updated, let's see if #14 works.

**Comment by ronaldtse** [comment]
Closed in #14 .
